server/app.py

```diff
@@ -132,6 +135,12 @@ def get_index():
     return {"status": "ok"}
 
 
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(request: Request, exc: RequestValidationError):
+    logger.error(str(exc))
```

Comment:
nit: can you add a bit more context to this log? Just like a "Request validation failed: %s"

Reply:
done